### PR TITLE
Fix BagBatcher for apollo

### PR DESCRIPTION
**Pull request by zurk**
Also, need to fix Uast2quant. Will be another PR.

### CODE REVIEW

**Comment by zurk** [comment]
Related to https://github.com/src-d/apollo/issues/18

File: sourced/ml/__main__.py [hunk 1]
```diff
@@ -135,13 +136,18 @@ def add_default_args(my_parser):
...
-    train_parser._actions = swivel.flags._global_parser._actions
-    train_parser._option_string_actions = \
-        swivel.flags._global_parser._option_string_actions
+
+    types = {"string": str,
+             "int": int,
+             "float": float,
+             "bool": bool}
+
+    for flag in swivel.FLAGS.__dict__['__wrapped'].__dict__['__flags'].values():
```
Comment (vmarkovtsev) [inline sourced/ml/__main__.py]: single quotes

File: sourced/ml/__main__.py [hunk 2]
```diff
@@ -135,13 +136,18 @@ def add_default_args(my_parser):
...
     train_parser.set_defaults(handler=run_swivel)
-    del train_parser._action_groups[train_parser._action_groups.index(
-        train_parser._optionals)]
-    train_parser._optionals = swivel.flags._global_parser._optionals
-    train_parser._action_groups.append(train_parser._optionals)
-    train_parser._actions = swivel.flags._global_parser._actions
-    train_parser._option_string_actions = \
-        swivel.flags._global_parser._option_string_actions
+
+    types = {"string": str,
```
Comment (vmarkovtsev) [inline sourced/ml/__main__.py]: Please make single line, I doubt that it will ever change

**Comment by bzz** [comment]
@vmarkovtsev all feedback seems to be addressed, might be ready from another round of reviews

**Pull request merged**